model_analyzer/result/model_config_measurement.py

```diff
@@ -32,11 +32,13 @@ class ModelConfigMeasurement:
     RunConfig run
     """
 
-    def __init__(self, model_config_name, model_specific_pa_params,
+    def __init__(self, model_config_name, model_name, model_specific_pa_params,
```

Comment:
Why does the model name need to be passed around? It seems to be used to index into the new ConstraintHolder class to get out that model's constraints, but can't we just pass around a single model's constraints instead of the model name + all constraints?